assets/js/utils.js

```diff
@@ -91,3 +91,20 @@ export function errorBoundary(f) {
     console.error(error);
   }
 }
+
+/**
+ * Joins arrays with a separator and returns the resulting array.
+ *
+ * @param {Array} array - The array that will have it's elements joined by the
+ *    provided separator.
+ *
+ * @param {*} separator - The separator that will be inserted between all the
+ *    elements in the provided array.
+ *
+ */
+export function arrayJoin(array, separator) {
+  return array.reduce((result, value, index) => {
+    return result.concat(value, separator);
+  }, []).slice(0, -1);
+}
+
```

Comment:
sorry man, I was vague before.

Isn't this what you want?

```
["bob", "dave", "lala"].join("&nbsp;")
"bob&nbsp;dave&nbsp;lala"
```

Reply:
I had to keep it in an array due to the other elements in the array being objects that are turned into elements. Essentially this is an array of HTML elements that are appended to an element in the DOM.

Reply:
ahhh cool - I didn't read your very clear doc - sorry